swagger.yaml

```diff
@@ -18,17 +18,28 @@ consumes:
 produces:
   - text/plain
 paths:
-  /:
+  /getSpendingAuthorit:
     get:
       summary: Get information
       description: Build and runtime information about the application.
```

Comment:
The summary and description are inaccurate for this endpoint as well. Can you change them as well? Also, can you add `tag` and `operationId` for this method as well?